man/systemd-vmspawn.xml

```diff
@@ -243,6 +243,16 @@
           <xi:include href="version-info.xml" xpointer="v256"/></listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><option>--trim-disk=</option><replaceable>BOOL</replaceable></term>
```

Comment:
hmm, Linux typically calls the concept "discard" afaics, not "trim"?

I think it would make sense to stick to that naming.